Doppler.Sap/Factory/ISapTaskHandler.cs

```diff
@@ -9,5 +10,6 @@ public interface ISapTaskHandler
         Task<SapBusinessPartner> TryGetBusinessPartner(int userId, string cuit, int userPlanTypeId);
         Task<SapBusinessPartner> TryGetBusinessPartnerByCardCode(string cardCode);
         Task<SapTask> CreateBusinessPartnerFromDopplerUser(SapTask task);
+        Task<SapSaleOrderInvoiceResponse> TryGeInvoiceByInvoiceId(int invoiceId);
```

Comment:
Is it on purpose?

```suggestion
        Task<SapSaleOrderInvoiceResponse> TryGetInvoiceByInvoiceId(int invoiceId);
```

Reply:
Sorry, it was typo. The correct is: ```Task<SapSaleOrderInvoiceResponse> TryGetInvoiceByInvoiceId(int invoiceId);``` 

I will change it.

Reply:
Done